src/components/UserUpdate/UserUpdate.test.tsx

```diff
@@ -100,31 +122,34 @@ describe('Testing User Update', () => {
             <UserUpdate {...props} />
           </Router>
         </I18nextProvider>
-      </MockedProvider>,
+      </MockedProvider>
     );
 
     await wait();
 
     userEvent.type(
       screen.getByPlaceholderText(/First Name/i),
-      formData.firstName,
+      formData.firstName
     );
     userEvent.type(
       screen.getByPlaceholderText(/Last Name/i),
-      formData.lastName,
+      formData.lastName
     );
     userEvent.type(screen.getByPlaceholderText(/Email/i), formData.email);
-    userEvent.selectOptions(screen.getByTestId('applangcode'), 'Français');
+    // userEvent.selectOptions(screen.getByTestId('applangcode'), 'Français');
```

Comment:
Why have you commented this line in tests

Reply:
Sorry that was not intended.